modules/microshift-greenboot-health-check-log.adoc

```diff
@@ -0,0 +1,38 @@
+// Module included in the following assemblies:
+//
+// * microshift_running applications/microshift-greenboot.adoc
+
+:_content-type: PROCEDURE
+[id="microshift-greenboot-access-health-check_{context}"]
+= Accessing health check output in the system log
```

Comment:
This sort of information should really go over in the RHEL docs, on a page like https://access.redhat.com/documentation/en-us/red_hat_enterprise_linux/9/html/composing_installing_and_managing_rhel_for_edge_images/managing-rhel-for-edge-images_composing-installing-managing-rhel-for-edge-images#rolling_back_rhel_for_edge_images

Reply:
@dhellmann If Greenboot is part of MicroShift, users should be able to stay in one set of docs to use it. Aside from that, the RHEL team has to evaluate added content.

Reply:
Greenboot is not really a MicroShift feature. MicroShift integrates with the Greenboot feature of RHEL.

Reply:
I hear that. I still think they'll look to these docs for information on how to look in the logs. Shall we leave in until we see content in RHEL on Greenboot/MicroShift?